### PR TITLE
WIP issue #13

### PR DESCRIPTION
**Pull request by lukko**
- Added method to update warehouse
- Refresh warehouse when new item is added to a container

_Feedback please... :)_

Issue #13

### CODE REVIEW

File: src/models/classes/container.js [hunk 1]
```diff
@@ -13,6 +13,10 @@ class Container extends StorageCore
         }
 
         super.addItem(product);
+
+        // Refresh warehouse
+        var warehouseController = new WarehouseController();
+        warehouseController._updateContainer();
```
Comment (N-Wouda) [inline src/models/classes/container.js]: I feel this part should be done in 'simulation' controller, _i.e._ in a part of the code that does not exists at this time. Similarly, there is also no code yet for updating the model/view when a new order arrives.

Requesting a specific controller to update the view from the model doesn't feel MVC to me, as this part will most likely only be called from such a 'simulation' controller, which is also in control of the view.

Comment (lukko) [reply src/models/classes/container.js]: Okay.

What about the rest?

**Comment by N-Wouda** [comment]
Looks good to me, aside from the one remark I left above.

**Comment by lukko** [comment]
Will remove it.

**Pull request merged**